fuel/app/classes/controller/welcome.php

```diff
@@ -29,7 +29,8 @@ class Controller_Welcome extends Controller
 	 */
 	public function action_index()
 	{
-		return Response::forge(View::forge('welcome/index'));
+      $hoge='fugafu';
+      return Response::forge(View::forge('welcome/index'));
```

Comment:
__[development][code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed